packages/core/test/lib/base.test.ts

```diff
@@ -571,7 +547,6 @@ describe('BaseClient', () => {
       expect(TestBackend.instance!.event!).toEqual({
         breadcrumbs: [normalizedBreadcrumb, normalizedBreadcrumb, normalizedBreadcrumb],
         contexts: normalizedObject,
-        environment: 'production',
```

Comment:
These test changes seen unrelated to tracestate

Reply:
See my comment above re: setting default environment earlier.